adal/src/main/java/com/microsoft/aad/adal/Oauth2.java

```diff
@@ -564,7 +572,7 @@ private AuthenticationResult postMessage(String requestMessage, Map<String, Stri
         }
         return result;
     }
-    
+
```

Comment:
nit: extra line

Reply:
If you view the expanded file this is a line break between methods